added tags to items for compatibility with tooltip rareness mod

will-da-pro opened the pull request.

Autovw commented:
Thanks for the addition, but you'll have to add them in the tags data generator as they will be removed during compilation otherwise.

will-da-pro commented:
Ok thanks, I added the tags to the tags generator. I also replaced ConventionalItemTag in the fabric item tags section with ItemTag, because ConventionalItemTag was deprecated and marked for removal.

Autovw commented:
I've made a couple changes, should be good to go now.

Thanks for the pull request!
